docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
```

Comment:
@mcruzdev the docker-compose file should have both.. the monolith and the database.. :) does that make sense? Do you know how to build the dockerfile from the app? or how to consume it from the external registry which is automatically published?

Reply:
Yeas, that make a sense. 
I know how to do this.

I will create a docker image using `docker build`, but will be a local image, is it okay?

Each developer that wants run the monolith will need to generates a image and depending of selected image tag... he will need to modify the docker-compose.yml.